.github/scripts/check-tag-format.sh

```diff
@@ -5,26 +5,24 @@ is_pre_release=$1
 current_tag=$(echo $GITHUB_REF | cut -d '/' -f 3 | tr -d ' ',v)
 
 if [ $is_pre_release = false ]; then
-  # Works with the format vX.X.X
+  # Works with the format vX.X.X, X being numbers
   #
   # Example of correct format:
   # v0.1.0
-  echo "$current_tag" | grep -E "[0-9]*\.[0-9]*\.[0-9]*$"
+  echo "$current_tag" | grep -E "^[0-9]+\.[0-9]+\.[0-9]+$"
   if [ $? != 0 ]; then
     echo "Error: Your tag: $current_tag is wrongly formatted."
     echo 'Please refer to the contributing guide for help.'
     exit 1
   fi
   exit 0
 elif [ $is_pre_release = true ]; then
-  # Works with the format vX.X.X-xxx-beta.X
-  # none or multiple -xxx are valid
+  # Works with the format vX.X.X-**.X, X being numbers
   #
   # Examples of correct format:
-  # v0.1.0-beta.0
-  # v0.1.0-xxx-beta.0
-  # v0.1.0-xxx-xxx-beta.0
-  echo "$current_tag" | grep -E "[0-9]*\.[0-9]*\.[0-9]*-([a-z]*-)*beta\.[0-9]*$"
+  # 0.2.0-pagination.1
+  # 0.2.0-v0.30.0-pre-release.0
```

Comment:
pre-release, do you mean `rc`, like `v0.30.0rc0`?

Reply:
Yes! do you think we should add it in the version name?
```
0.2.0-v0.30.0rc0-pre-release.0
```

Reply:
Oh ok, got it I've just understood which number correspond to which "project"